bundle/jsky.app.ot/src/main/java/jsky/app/ot/gemini/editor/sitequality/SiteQualityPanel.java

```diff
@@ -296,7 +301,9 @@ public void updateEnabledState(boolean enabled) {
     private static final long MS_PER_MINUTE = MS_PER_SECOND * 60;
     private static final long MS_PER_HOUR = MS_PER_MINUTE * 60;
 
-    private static final DateTimeFormatter dateFormat = DateTimeFormatter.ofPattern("yyyy-MM-dd HH:mm:ss z").withZone(ZoneId.of("UTC"));
+    private static final DateTimeFormatter dateFormat = DateTimeFormatter.ofPattern("yyyy-MMM-dd HH:mm:ss z")
+            .withZone(ZoneId.of("UTC"));
+    public static final Comparator<String> START_DATE_COMPARATOR = Comparator.comparingLong(s -> ZonedDateTime.parse(s, dateFormat).toEpochSecond());
```

Comment:
I think the `TimingWIndowTableModel` should provide the `Instant` in the call for `getValueAt` instead of a formatted `String` and then we wouldn't need a special comparator that has to parse the formatted `String`.  Unfortunately that's a bigger change though so it's not a good idea to do it now.  Perhaps we could though use the *same* `dateFormat` used by the `TimingWindowTableModel` so that there isn't a hidden dependency between the sorter and the formatter.  We'd need to extract the `DateTimeFormatter` from `TimingWindowTableModel` so it could be used by both.